lib/websocket_proxy.rb

```diff
@@ -51,7 +63,7 @@ def transmit(sockets, is_ws)
       @driver.parse(data)
     else
       @right.fetch(64.kilobytes) do |data|
-        @driver.binary(data)
+        @driver.frame(data)
```

Comment:
Can you please explain this?

Reply:
```
@driver.binary(data)
```
forcibly threats data obtained as `:binary` ([source code](https://github.com/faye/websocket-driver-ruby/blob/master/lib/websocket/driver/hybi.rb#L127-L129)) while

```
@driver.frame(data)
```
lets underlying library, websocket-driver, probe the type (see [this line](https://github.com/faye/websocket-driver-ruby/blob/master/lib/websocket/driver/hybi.rb#L166)). In the first case our `uint8utf8` data gets spoiled because it's intepreted incorrectly - should be interpreted as `:text` but is forcibly interpreted as `:binary`.

Reply:
:+1: